src/render.rs

```diff
@@ -70,7 +70,7 @@ impl<'a> Renderer<'a> {
     fn lookup_variable(&self, key: &str) -> TeraResult<Value> {
         // Look in the plain context if we aren't in a for loop
         if self.for_loops.is_empty() {
-            return self.context.lookup(key).cloned()
+            return self.context.pointer(&("/".to_string() + &key.replace(".", "/"))).cloned()
```

Comment:
I think this would look nicer, and avoid an allocation, if split into:

```rust
let path = &["/", &key.replace(".", "/")].join("");
return self.context.pointer(path).cloned()
    .ok_or_else(|| FieldNotFound(key.to_string()));
```